velox/type/tests/TypeTest.cpp

```diff
@@ -251,6 +251,27 @@ TEST(TypeTest, parseStringToDate) {
   EXPECT_EQ(parseDate("2135-11-09").days(), 60577);
 }
 
+TEST(TypeTest, dateFormat) {
+  auto parseDate = [](const std::string& dateStr) {
+    Date returnDate;
+    parseTo(dateStr, returnDate);
+    return returnDate;
+  };
+
+  EXPECT_EQ(
+      fmt::format("{}", parseDate("2015-12-24")), std::string("2015-12-24"));
```

Comment:
Is it necessary to call std::string for "2015-12-24" ? If not, let's remove to make the test shorter and easier to read.